src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -92,4 +101,39 @@ class WorkflowManagerActor extends Actor with CromwellActor {
   private def idByWorkflow(workflow: WorkflowActorRef): Option[WorkflowId] = {
     workflowStore.toMap collectFirst {case (k, v) if v == workflow => k}
   }
+
+  private def restartIncompleteWorkflows(): Unit = {
+    // If the clob inputs for this workflow can be converted to JSON, return the JSON
+    // version of those inputs in a Some().  Otherwise return None.
+    def clobToJsonInputs(result: QueryWorkflowExecutionResult): Option[binding.WorkflowRawInputs] = {
+      result.wdlRawInputs.parseJson match {
+        case JsObject(rawInputs) => Option(rawInputs)
+        case x =>
+          log.error(s"Error restarting workflow ${result.workflowId}: expected JSON inputs, got '$x'.")
+          None
+      }
+    }
+    val RestartableStates = Some(Seq(WorkflowSubmitted, WorkflowRunning))
```

Comment:
Normally the Caps go in the companion object (i.e. static constants), this is just a standard val